.github/workflows/translation_check.py

```diff
@@ -0,0 +1,103 @@
+import filecmp
+
+languages = ['ar-EG', 'ca-ES', 'cs-CZ', 'da-DK', 'de-DE', 'en-US', 'eo-OO', 'es-ES', 'fi-FI', 'fr-FR', 'hu-HU', 'it-IT',
+             'ja-JP', 'ko-KR', 'nb-NO', 'nl-NL', 'pl-PL', 'pt-BR', 'ru-RU', 'sv-SE', 'tr-TR', 'zh-CN', 'zh-TW']
+
+
+def file_to_dict(filename):
+    with open(filename, encoding="utf8") as f:
+        content = f.readlines()
+    lines = [line.strip() for line in content]
+    translations = {}
+    for line in lines:
+        if not line.startswith('#') and line.find(':') != -1:
+            split = line.split(':')
+            key = split[0].strip()
+            value = ':'.join(split[1:]).strip()
+            translations[key] = value
+
+    return translations
+
+
+def count_translations(dir_with_translations, print_info):
+    en_gb = file_to_dict('OpenRCT2/data/language/en-GB.txt')
+
+    missing_counters = dict.fromkeys(languages, 0)
+    same_counters = dict.fromkeys(languages, 0)
+    not_in_base_counters = dict.fromkeys(languages, 0)
+
+    for lang in languages:
+        translations = file_to_dict(dir_with_translations + '/' + lang + '.txt')
+        for base_string in en_gb:
+            if base_string not in translations:
+                missing_counters[lang] += 1
+                if print_info:
+                    print('[{}] Missing translation: {}'.format(lang, base_string))
```

Comment:
I guess using f-strings look clearer
```suggestion
                    print(f'[{lang}] Missing translation: {base_string}')
```

This would apply to all `format` calls in the patch

Reply:
I've suspected something like this exists in Python, but couldn't find it.